src/m2k_impl.cpp

```diff
@@ -400,3 +402,71 @@ bool M2kImpl::getLed()
 	}
 	return on;
 }
+
+std::vector<double> M2kImpl::readCalibrationLine(double temperature)
```

Comment:
This method should be refactored to show what it does  - something like:

getCalibrationValuesFromLUT()

Reply:
Modified.